lib/gateway/Shard.js

```diff
@@ -570,20 +570,26 @@ class Shard extends EventEmitter {
                             me: packet.d.user_id === this.client.user.id
                         };
                     }
+                } else {
+                    message = {
+                        id: packet.d.message_id,
+                        channel: channel || {id: packet.d.channel_id}
+                    };
+
+                    if(packet.d.guild_id) {
+                        message.guildID = packet.d.guild_id;
+                    }
                 }
                 /**
                 * Fired when someone adds a reaction to a message
                 * @event Client#messageReactionAdd
-                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel` keys. If the channel is not cached, channel key will be an object with only an id. No other property is guaranteed
+                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel` keys. If the channel is not cached, channel key will be an object with only an id. `guildID` will be present if the message was sent in a guild channel. No other property is guaranteed
```

Comment:
Probably better if the `guildID` info is in the same sentence as the message not cached instead of after

Reply:
`guildID` would be optional which is why I didnt put it in the same sentence.
How about "with `id`, `channel` and optionally `guildID` keys"?

Reply:
"with `id`, `channel`, and `guildID`, if in guild, keys" probably but I don't think that's much better haha
I just think "optionally" doesn't really fit the description and what you have right now feels out of place. My opinions though

Reply:
`id`, `channel`, and if inside a guild, `guildID` keys"
That's probably better

Reply:
Yeah I couldnt really think of a good way to phrase it haha. Ill change it to what you said.